### DISCUSSION TITLE
Make it flag users with swatsikas in their name

### DISCUSSION THREAD

**Discussion by a2937** [ThioJoe/YT-Spammer-Purge#723] [Ideas]
Quite a few bad actors on channels use the swastika as a symbol of them being racist or whatever. Perhaps we could work on detecting them as well. [Example of bad actor from a heavily hit channel](https://imgur.com/a/sIZ2fIq)

**Comment by a2937** [comment]
While thinking about it; possibly ones with fake verified checkmarks too.

**Comment by ThioJoe** [comment]
Can you link directly to the comment so I can get the character they're using

**Reply by a2937** [reply]
Here's a link to one of the channels. The comment I saw seems to have disappeared though. I can still see in the replies. 

[Screenshot of channel](https://imgur.com/a/r7id1tY) 

 https://www.youtube.com/channel/UCU99cS4hPU8P3-WVmo5v3KQ

**Reply by a2937** [reply]
This screenshot was the comment section of the Jacksepticeye Youtube channel, who like several other Youtubers are affected by the bot spam.

**Comment by WSrujan** [comment]
swatsikas (it means peace tho its often misunderstood) are not evil holocaust (reversed original used by hitter) is

To solve this problem show comment as separate so creator have to check content of comment to delete/flag.

**Reply by Firecul** [reply]
The differences between swastika 卐 (sun iirc) and sauwastika 卍 (night, again iirc) have been lost, I don't recall ever seeing it used on the general internet correctly. Certainly, I've never seen a commenter on YT use it while not referring to a certain party in WW2.

**Comment by GaviLazan** [comment]
Adding to this, having a "hate speech" mode (with/seperate from the spam mode) would be very helpful. The channel I manage has a little bit of spambots - crypto, whatsapp scams, etc - but very very often has real people leaving hate speech (racist, antisemitic, the like) on videos as well as groups of people coming from 4chan or bitchute brigading a video.

It's a totally different kind of spam, but one that YT is also very bad at detecting and removing. I'd argue that it's also possibly more harmful than annoying crypto and 18+ spambots.
